deepcell_label/blueprints_test.py

```diff
@@ -4,8 +4,9 @@
 
 import pytest
 import numpy as np
+from skimage.external.tifffile import TiffWriter
```

Comment:
Changed to tifffile. I had to pin the tifffile version to 2020.9.3 because that's the last version to support Python 3.6, but I'll update that in another PR to fully drop support for Python 3.6